be/src/Project/Services/Content/Content.Api/Dockerfile

```diff
@@ -2,6 +2,8 @@ FROM node:18-alpine
 
 RUN apk update && apk add  build-base gcc autoconf automake zlib-dev libpng-dev nasm bash vips-dev
 
+RUN npm install -g node-gyp
```

Comment:
**suggestion:** why do we have it here? I'm pretty much sure it's redundant

Reply:
I need this line of code because when I run project without it i have this error
![Zrzut ekranu 2023-10-05 081509](https://github.com/dawiddworak88/Giuru/assets/112848485/cc9e77f5-ea42-4843-bf23-8062d6930576)